aten/src/ATen/test/NamedTensor_test.cpp

```diff
@@ -43,6 +43,20 @@ TEST(NamedTensorTest, isNamed) {
   ASSERT_TRUE(tensor.is_named());
 }
 
+static bool dimnames_equal(at::DimnameList names, at::DimnameList other) {
```

Comment:
a dimnamelist is just a std::vector<Dimname> right? does std::vector<Dimname>==() do the same thing?

Reply:
a `DimnameList` is an `ArrayRef<Dimname>`. I haven't defined `operator==(Dimname,Dimname)` so we can't compare them directly. I'm not sure what I want the semantics of that to be yet.

Reply:
ah, right, I forgot there's some complexity with equality of name vs tag.